include/spdk/nvme.h

```diff
@@ -1141,6 +1142,36 @@ int spdk_nvme_ns_cmd_reservation_report(struct spdk_nvme_ns *ns,
 					void *payload, uint32_t len,
 					spdk_nvme_cmd_cb cb_fn, void *cb_arg);
 
+/**
+ * \brief Sends an abort command to the Nvme device for the request indicated
+ *  by cid and sqid
+ *
+ * \param ctrlr controller to the Nvme device
+ * \param cid identification number of the command to be aborted.
+ * \param sqid submission queue number of the command to be aborted
```

Comment:
The sqid parameter was changed to a qpair pointer, so this documentation needs to be updated.